### PR TITLE
Download JSON traces from Zipkin

### PR DESCRIPTION
**Pull request by cvetkovic**
Implemented option to download JSON traces directly from Zipkin in trace visualizer tool.
Plot visualization has a toggle for performance reasons.

### CODE REVIEW

File: tools/trace-plotter/main.go [hunk 1]
```diff
@@ -54,21 +58,93 @@ func main() {
...
+
+func downloadTraces(traces []*Trace) {
+	if _, err := os.Stat(*download); os.IsNotExist(err) {
+		err := os.MkdirAll(*download, 0700)
+		if err != nil {
+			panic(err)
+		}
+	}
+
+	fmt.Println("Downloading traces")
```
Comment (sk1tter) [inline tools/trace-plotter/main.go:95]: Maybe use logging?

**Review by sk1tter — APPROVED** [review]
LGTM

File: tools/trace-plotter/main.go [hunk 2]
```diff
@@ -54,21 +58,93 @@ func main() {
...
+		}
+		f, _ := os.Create(*fileName)
+		if err := page.Render(io.MultiWriter(f)); err != nil {
+			log.Errorf("Error rendering plot: %s", err)
+		}
+	}
+
+	if *download != "" {
+		downloadTraces(parsedE2ETraces)
+	}
```
Comment (ustiugov) [inline tools/trace-plotter/main.go:84]: Coding style suggestion

```suggestion
        if *download != "" {
		downloadTraces(parsedE2ETraces)
	}
	if *skipPlotting {
	    return
	}
	page := components.NewPage()
	page.PageTitle = "Trace Plots"

	parsedSystemTraces, systemDurations := ParseTraces(traces, "system")

	page.AddCharts(
		PlotGraph(parsedE2ETraces, e2eDurations, *zipkinURL, "e2e"),
		PlotGraph(parsedSystemTraces, systemDurations, *zipkinURL, "system"),
	)

	if filepath.Ext(*fileName) != ".html" {
		*fileName += ".html"
	}
	f, _ := os.Create(*fileName)
	if err := page.Render(io.MultiWriter(f)); err != nil {
		log.Errorf("Error rendering plot: %s", err)
	}
```

File: tools/trace-plotter/main.go (hunk 1 above)
Comment (ustiugov) [reply tools/trace-plotter/main.go:95]: yes for sure

**Review by ustiugov — CHANGES_REQUESTED** [review]
some coding style corrections needed

**Comment by cvetkovic** [comment]
Fixed all the comments.

**Review by ustiugov — APPROVED** [review]

**Pull request merged**